### DISCUSSION TITLE
🚀 Start UI [web] is now using React Hook Form

### DISCUSSION THREAD

**Discussion by yoannfleurydev** [BearStudio/start-ui-web#469] [Announcements]
<img width="1280" alt="Thumbnail Web - RHF announcement" src="https://github.com/BearStudio/start-ui-web/assets/3920615/72141428-6db9-4514-bf67-d5961f621481">

We are happy to announce that 🚀 Start UI [web] is now using React Hook Form instead of Formiz.

Formiz is a form library that we started to develop years ago at BearStudio because we often wanted simple, composable and multi-step forms.

Today, with 🚀 Start UI [web], we want to provide a full typed stack to our clients and users, so we decided to go with React Hook Form and zod resolvers to improve our quality on that part.
We are really proud of that new feature, we hope you'll love it! 

Feel free to check it out, share it to the world. We will be happy to have your feedbacks on that.

**Comment by oussama-gadria** [comment]
Congratulations on the exciting upgrade to 🚀 Start UI. Looking forward to trying out the fully typed stack and enjoying the enhanced form functionality. Great job, BearStudio!
